### PR TITLE
Feature/allow multi site

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@saeedmb8 @nathanbegbie ready for review

File: molo/surveys/tests/test_views.py [hunk 1]
```diff
@@ -248,14 +270,67 @@ def test_multi_step_multi_submissions_anonymous(self):
         self.assertContains(response,
                             'You have already completed this survey.')
 
-    def test_survey_template_tag_on_home_page(self):
+    def test_survey_template_tag_on_home_page_specific(self):
+        molo_survey_page, molo_survey_form_field = \
+            self.create_molo_survey_page(parent=self.surveys_index)
+        response = self.client.get("/")
+        self.assertContains(response,
+                            'Take The Survey</a>'.format(
```
Comment (nathanbegbie) [inline molo/surveys/tests/test_views.py]: .format unecessary?

File: molo/surveys/tests/test_views.py [hunk 2]
```diff
@@ -248,14 +270,67 @@ def test_multi_step_multi_submissions_anonymous(self):
...
+                                molo_survey_page.url))
+        self.assertContains(response, molo_survey_page.intro)
+        user = User.objects.create_superuser(
+            username='testuser', password='password', email='test@email.com')
+        self.client2.login(user=user)
+        response = self.client2.get(self.site2.root_url)
+        self.assertNotContains(
+            response, 'Take The Survey</a>'.format(molo_survey_page.url))
```
Comment (nathanbegbie) [inline molo/surveys/tests/test_views.py]: same here with .format

**Comment by nathanbegbie** [comment]
+1

**Pull request merged**